app/konnect/gateway-manager/configuration/consumer-groups.md

```diff
@@ -3,91 +3,156 @@ title: Create Consumer Groups in Konnect
-
-    If you switched it to **Scoped**, the rate limiting would apply the plugin to only one service, route, or consumer.
-
-    By default, the plugin is automatically enabled when the form is submitted.
-    You can also toggle the **This plugin is Enabled** button to configure the plugin without enabling it.
-    For this example, keep the plugin enabled.
-1. Complete only the following fields with the following parameters:
-    * **Config.limit:** 5
-    * **Config.window_size:** 30
-    * **Config.window_type:** Sliding
-    * **Config.retry_after_jitter_max:** 0
-    * **Config.enforce_consumer_groups:** true 
-    * **Config.consumer_groups:** Bronze
-
-    Besides the above fields, there may be others populated with default values. For this example, leave the rest of the fields as they are.
-1. Click **Save**.
-
-## Configure rate limiting for consumer groups
-
-In this section, you will configure the Rate Limiting Advanced plugin to set the rate limit to 6 requests every 30 seconds only for consumers in the Bronze tier.
-
-1. In {{site.konnect_short_name}}, click **Gateway Manager** in the sidebar.
-1. Select the **default** control plane.
-1. From the menu, open **Consumers**, then click the **Consumer Groups** tab.
-1. Click the **Bronze** consumer group you just created.
-1. Click **Add Configuration**.
-1. In the dialog, complete only the following fields with the following parameters:
-    * **Window Size:** 30
-    * **Window Type:** Sliding
-    * **Limit:** 6
-    * **Retry After Jitter Max:** 0
-1. Click **Save**.
-
-You have now set up a Bronze tier consumer group with one consumer, Amal, assigned to the group. Whenever the consumer Amal sends requests, they can make up to six requests every 30 seconds before hitting the rate limit. All other consumers are limited to only five requests every 30 seconds. 
\ No newline at end of file
+
+With [consumer groups](/gateway/latest/key-concepts/consumer-groups/), you can scope plugins to specifically defined groups and a new plugin instance will be created for each individual consumer group, making configurations and customizations more flexible and convenient. 
+For all plugins available on the consumer groups scope, see the [Plugin Scopes Reference](/hub/plugins/compatibility/#scopes).
+
+## Create a consumer group in {{site.konnect_short_name}}
+
+The following creates a new consumer group:
+
+1. In Gateway Manager, go to a {{site.base_gateway}} control plane > **Consumers**.
+2. Open the **Consumer Groups** tab.
+3. Click **New Consumer Group**.
+4. Enter a group name and select consumers to add to the group.
+5. Click **Save**.
+
+On the consumer group's overview page, you can now add any supported plugin. 
+This plugin will apply to all consumers in the group.
+
+## decK example
+
+Let's look at a more complex example where you define three consumer groups for rate limiting consumers.
```

Comment:
```suggestion
Let's look at a more complex example where you define three consumer groups for rate limiting consumers. In this example, each consumer group represents a different tier of support, such as Free, Basic, and Premium. Each tier will have different rate limits. For example, the Free tier has lower rate limits than the Premium tier.
```
Just thought a bit of context about why we're configuring different consumer groups would help.